### PR TITLE
SAK-50323 announcementswc Allow super users to view announcements

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-50323

### CODE REVIEW

File: announcement/announcement-impl/impl/src/java/org/sakaiproject/announcement/impl/BaseAnnouncementService.java [hunk 1]
```diff
@@ -1271,15 +1272,20 @@ private List<String> getExcludedSitesFromTabs() {
 	    return l;
 	}
 
+	public Filter getMaxAgeInDaysFilter(int maxAgeInDays) {
+
+		ViewableFilter viewableFilter = new ViewableFilter(null, null, Integer.MAX_VALUE, this);
```
Comment (ottenhoff) [inline announcement/announcement-impl/impl/src/java/org/sakaiproject/announcement/impl/BaseAnnouncementService.java:1277]: this is a lot of announcements! What's a practical value? 1000?

Comment (adrianfish) [reply announcement/announcement-impl/impl/src/java/org/sakaiproject/announcement/impl/BaseAnnouncementService.java]: @ottenhoff How's that look now?

Comment (ottenhoff) [reply announcement/announcement-impl/impl/src/java/org/sakaiproject/announcement/impl/BaseAnnouncementService.java]: thanks!

**Review by ottenhoff — APPROVED** [review]

**Comment by adrianfish** [comment]
@ottenhoff Thanks for the review

**Pull request merged**